decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    @nameable = nameable
+    super()
+  end
+
+  def correct_name
+    @nameable
+  end
```

Comment:
- [ ] the correct_name method should return the result of the correct_name method of the `@nameable` which looks something like below:
```
    def correct_name
      @nameable.correct_name
    end
```

Reply:
Dear @mikethreels 
Thank you so much for the feedback and requiring for changes. The changes have been applied.